.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,10 +30,10 @@ https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options
 ### Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My code follows the ZFS on Linux code style requirements.
+- [ ] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
 - [ ] I have updated the documentation accordingly.
-- [ ] I have read the **CONTRIBUTING** document.
-- [ ] I have added tests to cover my changes.
+- [ ] I have read the [**CONTRIBUTING** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
```

Comment:
What do you think about switching CONTRIBUTING to lower case.  I don't see a good reason to keep it all upper now that it's linked.

Reply:
@behlendorf done.